package.json

```diff
@@ -56,6 +56,7 @@
     "@babel/preset-env": "^7.6.0",
     "@testing-library/jest-dom": "^4.1.0",
     "@testing-library/svelte": "^1.8.0",
+    "@types/tether": "^1.4.4",
```

Comment:
I didn't even know we had types for tether! We probably need to update this.

Reply:
That's it! But how to use them correctly, I don’t know, actually. I mean, now in the Rollup configuration there isn’t even a TS code processing plugin at all.